src/components/People.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { Link, useLocation, useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+
+import { usePeopleDispatch, usePeopleState } from '../store/PeopleContext';
+
+import { Person } from '../types';
+import { getFilteredPeople } from '../services/getFilteredPeople';
+
+export const People: React.FC = () => {
```

Comment:
Consider renaming the component to a PeopleTableRows to improve code readability